test/e2e/instrumentation/events.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package instrumentation
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	v1beta1 "k8s.io/api/events/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/onsi/ginkgo"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	eventRetryPeriod  = 1 * time.Second
+	eventRetryTimeout = 1 * time.Minute
+)
+
+var _ = ginkgo.Describe("[sig-instrumentation] Events", func() {
```

Comment:
Let's introduce SIGDescribe the same way we do in other packages, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/framework.go

Reply:
Done.